application/libraries/admin/quexmlpdf.php

```diff
@@ -2260,6 +2260,22 @@ public function create($questionnaire)
         $this->fillLastPageBackground();
     }
 
+    /**
+     * Apply global settings from LimeSurvey application
+     *
+     * @access public
```

Comment:
This annotation is not needed, it's already set to `public` with the keyword. :)

Reply:
removed :)